ground/src/main/java/com/google/android/ground/ui/common/BaseMapViewModel.kt

```diff
@@ -48,6 +55,13 @@ constructor(
   mapController: MapController
 ) : AbstractViewModel() {
 
+  private val cameraZoomSubject: @Hot Subject<Float> = PublishSubject.create()
+  val cameraZoomUpdates: Flowable<Float> = cameraZoomSubject.toFlowable(BackpressureStrategy.LATEST)
```

Comment:
Consider using `SharedFlow` for new event streams. Feel free to change in a follow-up if that causes too much churn in this PR.

Reply:
I'll file a bug to do it separately